app/controllers/agent/matching/ClientAlreadySubscribedController.scala

```diff
@@ -16,33 +16,30 @@
 
 package controllers.agent.matching
 
-import auth.agent.UserMatchingController
-import config.AppConfig
+import controllers.SignUpBaseController
+import controllers.agent.actions.{ConfirmedClientJourneyRefiner, IdentifierAction}
 import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
-import services.{AuditingService, AuthService}
+import play.api.{Configuration, Environment}
+import uk.gov.hmrc.play.bootstrap.config.AuthRedirects
 import views.html.agent.matching.ClientAlreadySubscribed
 
 import javax.inject.{Inject, Singleton}
-import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.Future
 
 @Singleton
-class ClientAlreadySubscribedController @Inject()(val auditingService: AuditingService,
-                                                  val authService: AuthService,
+class ClientAlreadySubscribedController @Inject()(identify: IdentifierAction,
                                                   clientAlreadySubscribed: ClientAlreadySubscribed)
-                                                 (implicit val ec: ExecutionContext,
-                                                  val appConfig: AppConfig,
-                                                  mcc: MessagesControllerComponents) extends UserMatchingController {
+                                                 (val config: Configuration, val env: Environment)
+                                                 (implicit mcc: MessagesControllerComponents) extends SignUpBaseController with AuthRedirects {
```

Comment:
```suggestion
                                                 (implicit mcc: MessagesControllerComponents) extends SignUpBaseController {
```
I don't think AuthRedirects is used anywhere in this controller, so should be fine to remove.
If it is removed, config and env can also go as they are only needed due to AuthRedirects